R/assert_is_unique_dt.R

```diff
@@ -0,0 +1,47 @@
+#' Are the rows of the input data.table unique?
+#'
+#' Checks to see if the rows of the input data.table are unique for each
+#' combination of the id columns.
+#'
+#' @param dt input data.table to check.
+#' @param id_cols character vector of id columns that uniquely identify each row
+#' of `dt`.
+#' @inheritParams assertive.types::assert_is_data.table
```

Comment:
What does this do?

Reply:
This needs to be removed, I initially set up to be identical to the `assertive` functions but thought the error messages were difficult to interpret.